main.tf

```diff
@@ -106,6 +115,15 @@ resource "aws_dynamodb_table" "this" {
     }
   }
 
+  dynamic "on_demand_throughput" {
+    for_each = length(var.on_demand_throughput) > 0 ? [var.on_demand_throughput] : []
+
+    content {
+      max_read_request_units  = try(on_demand_throughput.value.max_read_request_units, null)
```

Comment:
Hello,

as per documentation the default value should be -1 and no "null". Please check on:

[on_demand_throughput](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table#on_demand_throughput)

Reply:
no - @magreenbaum has written it correctly